### PR TITLE
Fix crowbar_register on SLE12

### PR DESCRIPTION
**Pull request by vuntz**
Do not install suse-cloud-release but suse-sle12-cloud-compute-release.

### CODE REVIEW

File: chef/cookbooks/provisioner/templates/suse/crowbar_register.erb [hunk 1]
```diff
@@ -182,7 +182,11 @@ PACKAGES_INSTALL="$PACKAGES_INSTALL parted lvm2 ipmitool curl tack nfs-client rs
 
 zypper --non-interactive install -t pattern $PATTERNS_INSTALL
 # Auto-agree with the license since it was already agreed on for the admin server
+<% if @os == "suse-11.3" -%>
```
Comment (dirkmueller) [inline chef/cookbooks/provisioner/templates/suse/crowbar_register.erb]: previously we used

<% if @target_platform_version == "11.3" -%>

**Comment by MaximilianMeister** [comment]
tested (although not within barclamp context, just `crowbar_register`)
+1

**Comment by tboerger** [comment]
Why do you install som gems only on sle < 12?

**Comment by dirkmueller** [comment]
+1

**Comment by vuntz** [comment]
> Why do you install som gems only on sle < 12?

I'm actually thinking of just removing that line as it's only needed when doing a hardware update, I think. But I went the safe way and left that for SLE11.

**Comment by tboerger** [comment]
So these gems are only used on older SLES versions?

**Comment by dirkmueller** [comment]
Improves things, so merging.

**Comment by vuntz** [comment]
@tboerger as I said, I think they're actually not used at all. But I don't feel comfortable merging this now. Let me add a new PR for post-tex.

**Comment by tboerger** [comment]
Too late, @dirkmueller already merged that

**Pull request merged**